README.md

```diff
@@ -5,6 +5,43 @@ to fully record the interactions
 between assets (files and tools)
 when a software artifact is being built (compiled).
 
+## Synopsis
+**build-recorder** [-o outfile] command
+* outfile: The output file, by default "build-recorder.out".
+* command: The original build command with all its arguments.
+    * e.g. cc -o hello helloworld.c 
+
+## Description
+**build-recorder** is a command line tool for linux 5.3+ that records
+information about build processes. It achieves this by running transparently
+in the background while the build process is running, tracing it
+and extracting all relevant information, which it then stores in the output
+file in RDF Turtle format.
+
+A complete schema for the generated RDF can be found in docs/output.md.
+
+**build-recorder** works regardless of the programming language, build system
+or configuration used. In fact there is no limitation as to what the supplied 
+command should be. If it runs, **build-recorder** can trace it.
+
+## Build
+To build it you are going to need the following tools:
+* A C compiler
+* make
+* autoconf
+* automake
+
+As well as the following libraries:
+* libcrypto
+
+### Build Instructions
```

Comment:
This is "build from the git repo".

A similar one would be "build from the released tarball" (without `autoreconf`).